### ISSUE TITLE
Agree on license for public content

### ISSUE THREAD

**Issue by 20zinnm** [ourdataourhands/non-profit#8]
I'm fine with MIT or CC Share-Alike 4.0.

**Comment by ivanstegic** [comment]
Good point. I would love the license to be as permissive as possible. @jprenken any thoughts on which one?

**Comment by 20zinnm** [comment]
As permissive as possible is CC-Zero or the [WTFPL](http://www.wtfpl.net/). I think the [GNU GPLv3](https://choosealicense.com/licenses/gpl-3.0/) might also be a good fit for this project, because we want any improvements to make their way back here, no?

**Comment by ivanstegic** [comment]
My original thought was GPL3, would love to see improvements make their way back, yes.

**Comment by ivanstegic** [comment]
I'd like to land on [GNU GPLv3](https://choosealicense.com/licenses/gpl-3.0/) -- @jprenken @andrewkwh please let me know if that's OK with you?
